waku/node/v2/config.nim

```diff
@@ -73,6 +73,21 @@ type
       defaultValue: ""
       name: "storenode" }: string
 
+    historicalnode* {.
```

Comment:
Sorry but these three ones seems worse to me, the previous one was simplistic but clear. Full node eg seems like a misnormer.

I'd make sure we know what we are doing and what we want before we increase the sophistication. KISS. Make sure we understand the problem well enough before writing a solution for it. See https://github.com/vacp2p/specs/issues/167 for context